developer-docs-site/docs/concepts/coin-and-token/Working with PropertyMap Off-Chain.md

```diff
@@ -0,0 +1,22 @@
+# Working with PropertyMap Off-Chain
+
+Token module uses property map [module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/property_map.move) to store on-chain properties of token. PropertyMap maps a string key to a property value on-chain, which stores the value in BCS serialized format and its type. Currently, only primitive types (bool, u8, u64, u128, address and String) are supported in property map. 
+
+With both the value and typing, property map can be used to read and write values of different types in a map data structure on-chain. 
+
+### Read and write property map using TS SDK
+
+Our SDK supports read and write property map from typescript directly. It saves you from knowing the details for BCS serialization. 
```

Comment:
I think you mean the property map can be read and written to?

Reply:
updated